commit/merkleroot/rmn/client.go

```diff
@@ -0,0 +1,19 @@
+package rmn
+
+import (
+	"context"
+	"errors"
+)
+
+// ErrTimeout is returned when the signature computation times out.
+var ErrTimeout = errors.New("signature computation timeout")
+
+// Client contains the methods required by the plugin to interact with the RMN nodes.
+type Client interface {
```

Comment:
Alright I see, so this Client encapsulates both ObservationRequest and ReportSignatureRequest? In that case can we make it clear how this interface's functions/behavior map to what is in the RMN DD?

Reply:
added a comment explaining this